src/SignhostAPIClient/Rest/SignhostApiReceiver.cs

```diff
@@ -50,8 +50,8 @@ public SignhostApiReceiver(SignhostApiReceiverSettings receiverSettings)
 			}
 
 			if (!string.IsNullOrEmpty(postbackTransaction.Id) &&
-				!string.IsNullOrEmpty(postbackChecksum)
-			) {
+				!string.IsNullOrEmpty(postbackChecksum))
+			{
```

Comment:
Sorry this doesn't match our coding-standards / guidelines, we require the openening `{` to be on the same line as the `)`, sometimes for readability we therefore choose to place the `) {` on a newline.

Reply:
Stylecop code analyzer rules and code guidelines differ from one another.
Dropped this commit.